docs/Project.toml

```diff
@@ -1,3 +1,4 @@
 [deps]
 AbstractDifferentiation = "c29ec348-61ec-40c8-8164-b8c60e9d9f3d"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
The question I always ask myself is: when we don't include `docs/Manifest.toml`, how is Julia supposed to know we want to `dev` the current code of the package into the docs environment instead of downloading the stable version from the general registry?

Reply:
Well, you specify it 😅 That's exactly what julia-docdeploy (https://github.com/julia-actions/julia-docdeploy/blob/580a86b97cef20258d96b53292326e6258dd6859/action.yml#L30) and basically every custom Julia doc build does.

Reply:
for CI I understand, but locally that assumes the developer already has a working manifest, otherwise when they activate and instantiate the docs environment that will by default download the stable version

Reply:
Typically, locally you also first `dev` the package. I have really bad experiences with committing Manifest files for docs, so I'm strongly convinced that we should save ourselves the headaches that come with it and only add a Project.toml file.

Reply:
alright, I trust you
gonna look up how to add CompatHelper for a subfolder

Reply:
https://juliaregistries.github.io/CompatHelper.jl/dev/options/#Packages-in-subdirectories